server/mergin/sync/workspace.py

```diff
@@ -349,3 +351,21 @@ def project_access(self, project: Project) -> List[ProjectAccessDetail]:
                 )
                 result.append(member)
         return result
+
+    def server_editors_count(self) -> int:
+        if Configuration.GLOBAL_ADMIN or Configuration.GLOBAL_WRITE:
+            return User.query.filter(
+                is_(User.username.ilike("deleted_%"), False) | User.active,
+            ).count()
+
+        return (
+            db.session.query(ProjectUser.user_id)
```

Comment:
distinct?

Reply:
There should be group by and select just user_id